src/net/sourceforge/kolmafia/session/ChoiceAdventures.java

```diff
@@ -6221,6 +6221,15 @@ public static final Option findOption(final Option[] options, final int decision
         "Crimbo Train (Dining Car)",
         new Option("acquire 3 lost elf trunks", 1),
         new Option("decrease Trainbot strength", 2));
+
+    // Slagging Off
+    new ChoiceAdventure(
+        1489,
+        "Crimbo22",
+        "Crimbo Train (Coal Car)",
+        new Option("acquire crystal Crimbo Goblet", 1),
```

Comment:
Perhaps Crimbo goblet or Crimbo Platter?  The inconsistency jumps out at me and I think the game uses lower case.

Reply:
Should be Crimbo goblet. Will fix.